src/components/Icon/Icon.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
```

Comment:
This single Icon component helps enforce consistency in using Icons and is useful for showcasing the used icons in Storybook. It also makes the consuming code simpler, and easier to read since it's more semantic to see `<Icon/>`.

This also helps avoid adding lots of different components since you should add it here to "catalog" the icons the app uses. This little extra friction should help avoid people needlessly adding new icons when an existing icon may work best.

Reply:
We have a user story in process to create an icon component.